### ISSUE TITLE
unet.py中的疑问

### ISSUE THREAD

**Issue by haolidong** [cuilunan/Unet-sensing-image-tensorflow#1]
1. unet.py文件中的第183行没写完整，应该是 loss =       tf.reduce_mean((tf.nn.sparse_softmax_cross_entropy_with_logits(logits=logits, labels=annotation))
2. 请教，unet( )返回的conv10是[None, IMAGE_SIZE, IMAGE_SIZE, 2]，annotation是[None, IMAGE_SIZE, IMAGE_SIZE, 1]，上面的loss怎么计算呢？
